index.html

```diff
@@ -22,18 +21,15 @@
     'Reva Lingenfelter',
     'Myung Bramer',  
     'Renae Deason',  
-    'Josephine Sentell',
     'Rogelio Zahm',  
     'Anglea Haga',  
     'Deanna Pippen',  
     'Carissa Minjares',  
     'Elli Schecter',  
-    'Julianne Oconnell',  
-    'Julieann Richert',  
     'Chet Herter',  
     'Dorene Prestwood'
   ];
-
+names.sort();
```

Comment:
Huh..! I thought that wouldn't be possible because 'names' is const.

Reply:
Me neither! But I figured I might aswell try